services/fileuploader/gzwriter.go

```diff
@@ -0,0 +1,62 @@
+package fileuploader
```

Comment:
why is the rational behind creating this `services/fileuploader/gzwriter`? I mean it seems to be doing things very specific to our jobsDB backup (atleast as of now). Creating a gzFileHandler which has a map of path & gzipwriter. Where else are we going to use it? If no where, then won’t it be better to move it to jobsdb backup?

Reply:
The idea is it can be used anywhere not just jobsdb so I kept it in services folder